contracts/interfaces/Integrations.sol

```diff
@@ -97,14 +101,14 @@ interface ISafeProtocolPlugin is IERC165 {
     function name() external view returns (string memory name);
 
     /**
-     * @notice A funtion that returns version of the plugin
+     * @notice A function that returns version of the plugin
      * @return version string version of the plugin
      */
     function version() external view returns (string memory version);
 
     /**
-     * @notice A funtion that returns version of the plugin.
-     *         TODO: Define types of metadata provider and possible values of location in each of the cases.
+     * @notice A funtion that returns information about the type of metadata provider and its location.
+     *         For more information on metadata provider, refer to https://github.com/safe-global/safe-core-protocol-specs/tree/main/metadata#metadata-provider.
```

Comment:
This link will be dead if the branch name or readme structure changes. We should use permalinks instead.

Reply:
Thought the same but, if we use permalink it might point to updated info in future.
I think we can use this https://github.com/safe-global/safe-core-protocol-specs/tree/main/metadata that would handle any changes to structure of README and update manually later if needed

Reply:
There's no easy way to track dead/outdated links in code, and IMO pointing to outdated data is better than pointing to the dead one. You could also just link the repo without the section directly, just "For more information on metadata provided, refer to the protocol specs"